.github/pull_request_template.md

```diff
@@ -19,7 +19,14 @@ If this PR introduces a new feature, it is better to list some use cases here, a
 
 ## Checklist
 
-1. Pre-commit or other linting tools are used to fix the potential lint issues.
-2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
-3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
-4. The documentation has been modified accordingly, like docstring or example tutorials.
+**Before PR**:
+
+- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
+- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
+- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
```

Comment:
This is for new functionalities. Typos/docs does not need tests. Refactors are covered by existing tests. Bugs are listed int above item